NEOSPlus.sln

```diff
@@ -3,34 +3,44 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.1.32414.318
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NEOSPlus", "NEOSPlus\NEOSPlus.csproj", "{08A94620-ECB7-41FD-8C9C-C11F2EBFC776}"
```

Comment:
How the hell did your instance of Visual Studio manage to do this?

Reply:
visual studio is smoking agian.